docs/python/settings-reference.md

```diff
@@ -52,18 +52,22 @@ The language server settings apply when `python.languageServer` is `Pylance` or
 | --- | --- | --- |
 | typeCheckingMode | off | Specifies the level of type checking analysis to perform. Available values are `off`, `basic`, and `strict`. When set to `off` no type checking analysis is conducted; unresolved imports/variables diagnostics are produced. When set to `basic` non-type checking-related rules (all rules in `off`), as well as basic type checking rules are used. When set to `strict` all type checking rules at the highest severity of error (including all rules in `off` and `basic` categories) are used. |
 | diagnosticMode | openFilesOnly | Specifies what code files the language server analyzes for problems. Available values are `workspace` and `openFilesOnly`. |
+| include | [] | Paths of directories or files that should be included in analysis. If no paths are specified, Pylance defaults to the directory that contains the workspace root. Paths may contain wildcard characters such as `**` (a directory or multiple levels of directories), `*` (a sequence of zero or more characters), or `?` (a single character). |
```

Comment:
You might consider breaking up this text with `<br>` tags where you think there is a logical break in the sentences.

```md
Paths of directories or files that should be included in analysis.<br>If no paths are specified, Pylance defaults to the directory that contains the workspace root. <br>Paths may contain wildcard characters such as `**` (a directory or multiple levels of directories), `*` (a sequence of zero or more characters), or `?` (a single character).
```